Kudu.Contracts/Kudu.Contracts.csproj

```diff
@@ -142,6 +142,7 @@
     <Compile Include="Settings\DeploymentSettingsExtension.cs" />
     <Compile Include="Settings\IDeploymentSettingsManager.cs" />
     <Compile Include="Settings\SettingsKeys.cs" />
+    <Compile Include="SiteExtensions\InvalidEndpointException.cs" />
```

Comment:
Unless you plan to catch this explicitly and perform specific task, then I don't think this is necessary.   Simple trace in the code should do.